### DISCUSSION TITLE
Preview for data saved in Arrow IPC format?

### DISCUSSION THREAD

**Discussion by dmbates** [posit-dev/positron#3706] [Q&A] [answered]
I save data frames in Arrow IPC format so that I can access them easily (and unambiguously) in Python, R or Julia.  I have been searching for some time for a data preview plugin for VS Code that would display Arrow.  The only one I have found is Data Preview.  Are there recommendations for others?  Is the Arrow format of sufficient interest to Posit to include it in data previewers?

**Comment by jthomasmock** [comment, marked as answer]
Thanks for your interest in the Data Explorer -- currently we are relying on in-memory dataframes, eg R data.frame/tibble/data.table or Python: Pandas/polars dataframes.

We are interested in on-disk or Arrow data in the future, see: https://github.com/posit-dev/positron/issues/2186 if you'd like to upvote or add some additional context.

**Comment by wesm** [comment]
Just confirming we do plan to enable the Positron data explorer to be used for previewing Arrow IPC files (or Parquet files, or other common tabular data files), so stay tuned for that =)
